fixtures/infra.tf.json

```diff
@@ -1,88 +1,152 @@
 {
-  "output": {
-    "arn": {
-      "value": "${aws_dynamodb_table.basic-dynamodb-table.arn}"
+  "output": [
+    {
+      "arn": [
+        {
+          "value": "${aws_dynamodb_table.basic-dynamodb-table.arn}"
+        }
+      ]
     },
-    "endpoint": {
-      "value": "http://dynamodb.com:8080/endpoint/${aws_dynamodb_table.basic-dynamodb-table.arn}"
+    {
+      "endpoint": [
+        {
+          "value": "http://dynamodb.com:8080/endpoint/${aws_dynamodb_table.basic-dynamodb-table.arn}"
+        }
+      ]
     }
-  },
-  "provider": {
-    "aws": {
-      "access_key": "${var.FREY_AWS_ACCESS_KEY}",
-      "region": "us-east-1",
-      "secret_key": "${var.FREY_AWS_SECRET_KEY}"
+  ],
+  "provider": [
+    {
+      "aws": [
+        {
+          "access_key": "${var.FREY_AWS_ACCESS_KEY}",
+          "region": "us-east-1",
+          "secret_key": "${var.FREY_AWS_SECRET_KEY}"
+        }
+      ]
     }
-  },
-  "resource": {
-    "aws_dynamodb_table": {
-      "basic-dynamodb-table": {
-        "hash_key": "UserId",
-        "name": "GameScores",
-        "range_key": "GameTitle",
-        "read_capacity": 20,
-        "write_capacity": 20,
-        "attribute": [
-          {
-            "name": "TopScore",
-            "type": "N"
-          },
-          {
-            "name": "UserId",
-            "type": "N"
-          },
-          {
-            "name": "GameTitle",
-            "type": "S"
-          }
-        ],
-        "global_secondary_index": {
-          "hash_key": "GameTitle",
-          "name": "GameTitleIndex",
-          "non_key_attributes": [
-            "UserId"
-          ],
-          "projection_type": "INCLUDE",
-          "range_key": "TopScore",
-          "read_capacity": 10,
-          "write_capacity": 10
+  ],
+  "resource": [
+    {
+      "aws_dynamodb_table": [
+        {
+          "basic-dynamodb-table": [
+            {
```

Comment:
@kvz Since my PR has been merged (https://github.com/hashicorp/hcl/pull/149#issuecomment-244823992), I think we should be able to switch away from my private fork. Do you agree?